### ISSUE TITLE
List of questions about the Foundation from call 3

### ISSUE THREAD

**Issue by lrettig** [stacksgov/pm#11]
I promised to record and share these on yesterday's call.

**Comment by lrettig** [comment]
Thanks, I agree with this. Let me rephrase my question, then: an open issue generally indicates some work that needs to be done. What work is actually pending here, if any? :)

**Comment by joberding** [comment]
Most of these questions have been answered or are moot with the exception of a few like these:
- What will its role be in managing tech and tech standards, e.g., data structures? (primarily how)
-  Should it have formal working groups, akin to W3C/IETF?
- Does the community have a voice in its leadership? (primarily how?)
That's my .2 cents. What do you think?

**Comment by lrettig** [comment]
Thanks @joberding :) I think there are two categories of questions here. The first is, for the sake of accountability and transparency, where were certain decisions made? E.g., why was Delaware chosen as the jurisdiction? What does its initial funding look like?

The second category includes the questions you mentioned, which are more forwards-looking.

For the record I'm happy to keep this issue open, but I think, if the task here is/was "write a list of questions," that task is done and the list could be posted elsewhere. If on the other hand the task is "answer these questions", then we should discuss how to get those answers :)
